Hackerrank2.py

```diff
@@ -0,0 +1,15 @@
+# You are given the firstname and lastname of a person on two different lines.
+# Your task is to read them and print the following:
+# Hello firstname lastname! You just delved into python.
+
+
+def print_full_name(a, b):
+    print("Hello {0} {1}! "
+          "You just delved into python.".format(first_name, last_name))

```

Comment:
if you are in function, please use local variables a,b

Reply:
fixed